kpops/utils/docstring.py

```diff
@@ -1,51 +1,48 @@
 import logging
 
-from pydantic import BaseModel
-
 log = logging.getLogger("docstring_utils")
 
 
-def describe_attr(name: str, docstr: str) -> str:
+def describe_attr(name: str, docstr: str | None) -> str:
     """Read attribute description from class docstring
 
     Works only with reStructuredText docstrings.
 
     :param name: Attribute name
     :type name: str
     :param docstr: Docstring from which to read. Note that the class' docstring is stored in attr `__doc__`
-    :type docstr: str
+    :type docstr: str, None
     :returns: Description of the class attribute read from the class docstring
     :rtype: str
     """
-    docstr = docstr.partition(f":param {name}:")[2]
+    if not isinstance(docstr, str):
+        return ""
 
+    docstr = docstr.partition(f":param {name}:")[2]
     docstr = _trim_description_end(docstr)
 
     return docstr
```

Comment:
collapse to one line
```suggestion
    return _trim_description_end(docstr)
```